From 619968f8698a024debc4eb3ee9f694cdb92d12f2 Mon Sep 17 00:00:00 2001
From: hienhuynh <hien.huynh.px@renesas.com>
Date: Tue, 8 Mar 2022 14:24:23 +0700
Subject: [PATCH 4/8] Reduce MM_OMXBUF_SIZE for omx

This commit is created to reduce MM_OMXBUF_SIZE for RZ/G2L Series.

Signed-off-by: hienhuynh <hien.huynh.px@renesas.com>
---
 .../mmngr/mmngr-module/files/mmngr/include/mmngr_private.h      | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/mmngr_drv/mmngr/mmngr-module/files/mmngr/include/mmngr_private.h b/mmngr_drv/mmngr/mmngr-module/files/mmngr/include/mmngr_private.h
index 56ab1f6..e7c9d18 100644
--- a/mmngr_drv/mmngr/mmngr-module/files/mmngr/include/mmngr_private.h
+++ b/mmngr_drv/mmngr/mmngr-module/files/mmngr/include/mmngr_private.h
@@ -205,7 +205,7 @@ static void mm_exit(void);
 static int validate_memory_map(void);
 
 #ifdef MMNGR_SALVATORX
-#define MM_OMXBUF_SIZE		(256 * 1024 * 1024)
+#define MM_OMXBUF_SIZE		(128 * 1024 * 1024)
 #endif /* MMNGR_SALVATORX */
 
 #ifdef MMNGR_EBISU
-- 
2.17.1

